starter-code/src/contacts.js

```diff
@@ -0,0 +1,91 @@
+import React, { Component } from "react";
+import contacts from "./contacts.json";
+
+class Contacts extends Component {
+  state = {
+    contacts: contacts.slice(0, 5)
+  };
+
+  randomContact = () => {
+    let randCeleb =
+      contacts[Math.floor(Math.random() * (contacts.length - 6)) + 6];
+    return randCeleb;
+  };
+
+  toggleContact = () => {
+    //contacts.splice(0, 5) + this.randomContact()
+
+    let newContacts = this.state.contacts;
+    newContacts.push(this.randomContact());
+
+    this.setState({
+      contacts: newContacts
+    });
+    console.log(this.randomContact());
+    console.log(this.state.contacts, "contacts");
+  };
+
+  toggleName = () => {
+    let sortedNames = this.state.contacts;
+    console.log(this.state.contacts);
+    sortedNames.sort((a, b) => (a.name > b.name ? 1 : -1));
+
```

Comment:
maybe here you could have return the reversed sorted array?
Right now is only sorting in ascending order.